src/catalog/src/information_schema/memory_table/tables.rs

```diff
@@ -227,6 +227,191 @@ pub fn get_schema_columns(table_name: &str) -> (SchemaRef, Vec<VectorRef>) {
             vec![],
         ),
 
+        // TODO: GreptimeDB already has query tracing, support later. cc @Taylor-lagrange
```

Comment:
I posted an issue to track this, we can remove this TODO from the comment. https://github.com/GreptimeTeam/greptimedb/issues/3064
```suggestion
```